Samples/Samples/View/CalendarPage.xaml.cs

```diff
@@ -17,10 +17,10 @@ public CalendarPage()
         protected override void OnAppearing()
         {
             base.OnAppearing();
+			
+            var viewModel = BindingContext as CalendarViewModel;
 
-            var vm = BindingContext as CalendarViewModel;
-
-            vm?.RefreshCalendars();
+            // viewModel?.RefreshCalendars();
```

Comment:
do we need this enabled? for the timebeing unless it causes a crash we can live with the android permissions issue

Reply:
yeah we can enable it, must have accidentally been pushed through, it's my temporary work around to allow android work to go through